rustls/tests/api.rs

```diff
@@ -657,6 +662,16 @@ impl ResolvesServerCert for ServerCheckCertResolve {
             }
         }
 
+        if let Some(expected_cipher_suites) = &self.expected_cipher_suites {
+            if expected_cipher_suites != client_hello.cipher_suites() {
```

Comment:
Is there any reason not to use `assert_eq!()` here?

Reply:
Not really, I copied what was already there. I can change it and update others that fit the pattern as well.